packages/core/src/generators/init/generator.ts

```diff
@@ -3,13 +3,18 @@ import {
   readJson,
   readWorkspaceConfiguration,
   Tree,
+  updateWorkspaceConfiguration,
   WorkspaceConfiguration,
   writeJson,
 } from '@nrwl/devkit';
+import { DotNetClient, dotnetFactory } from '@nx-dotnet/dotnet';
 
 import { CONFIG_FILE_PATH, NxDotnetConfig } from '@nx-dotnet/utils';
 
-export default async function (host: Tree) {
+export default async function (
+  host: Tree,
+  dotnetClient = new DotNetClient(dotnetFactory()),
```

Comment:
Add a new parameter here between host + dotnetClient. I recommend 
```
export default async function (
  host: Tree,
  _: null,
  dotnetClient = new DotNetClient(dotnetFactory()),
) {
```

Nx CLI will call the function with 2 parameters (and the default dotnetClient), so we have to allocate the first two parameters to Nx.